lib/stimulus_reflex/broadcasters/page_broadcaster.rb

```diff
@@ -8,7 +8,7 @@ def broadcast(selectors, data)
 
       return unless page_html.present?
 
-      document = Nokogiri::HTML(page_html)
+      document = Nokogiri::HTML.parse(page_html)
```

Comment:
I changed this because it's easier to stub - I think it should be just an alias?